packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/tests/index.test.js

```diff
@@ -34,1310 +53,145 @@ describe('Admin | containers | ListView', () => {
-        left: -4px;
-        right: -4px;
-        border: 2px solid transparent;
-      }
+    const loadingElement = await screen.findByText('Loading content.');
 
-      .c11:focus-visible {
-        outline: none;
-      }
+    expect(loadingElement).toBeInTheDocument();
 
-      .c11:focus-visible:after {
-        border-radius: 8px;
-        content: '';
-        position: absolute;
-        top: -5px;
-        bottom: -5px;
-        left: -5px;
-        right: -5px;
-        border: 2px solid #4945ff;
-      }
+    await waitForElementToBeRemoved(() => screen.getByText('Loading content.'));
 
-      .c12 {
-        -webkit-text-decoration: none;
-        text-decoration: none;
-      }
+    expect(screen.getByText('http:://strapi.io')).toBeInTheDocument();
+  });
 
-      .c12[aria-disabled='true'] {
-        border: 1px solid #dcdce4;
-        background: #eaeaef;
-      }
+  it('should show a loader when permissions are loading', () => {
+    useRBAC.mockImplementation(() => ({
+      isLoading: true,
+      allowedActions: { canUpdate: true, canCreate: true, canDelete: true },
+    }));
 
-      .c12[aria-disabled='true'] .c14 {
-        color: #666687;
-      }
+    useNotification.mockImplementation(() => jest.fn());
 
-      .c12[aria-disabled='true'] svg > g,.c12[aria-disabled='true'] svg path {
-        fill: #666687;
-      }
+    const { getByText } = render(App);
 
-      .c12[aria-disabled='true']:active {
-        border: 1px solid #dcdce4;
-        background: #eaeaef;
-      }
+    expect(getByText('Loading content.')).toBeInTheDocument();
+  });
 
-      .c12[aria-disabled='true']:active .c14 {
-        color: #666687;
-      }
+  it('should show a list of webhooks', async () => {
+    useRBAC.mockImplementation(() => ({
+      isLoading: false,
+      allowedActions: { canUpdate: true, canCreate: true, canDelete: true },
+    }));
 
-      .c12[aria-disabled='true']:active svg > g,.c12[aria-disabled='true']:active svg path {
-        fill: #666687;
-      }
+    render(App);
 
-      .c12:hover {
-        border: 1px solid #7b79ff;
-        background: #7b79ff;
-      }
+    await waitFor(() => {
+      expect(screen.getByText('http:://strapi.io')).toBeInTheDocument();
+    });
+  });
 
-      .c12:active {
-        border: 1px solid #4945ff;
-        background: #4945ff;
-      }
+  it('should show confirmation delete modal', async () => {
+    useRBAC.mockImplementation(() => ({
+      isLoading: false,
+      allowedActions: { canUpdate: true, canCreate: true, canDelete: true },
+    }));
 
-      .c12 svg > g,
-      .c12 svg path {
-        fill: #ffffff;
-      }
+    const { container, getByText } = render(App);
+    await waitFor(() => {
+      screen.getByText('http:://strapi.io');
+    });
 
-      @media (prefers-reduced-motion:reduce) {
-        .c39:before {
-          -webkit-transition: none;
-          transition: none;
-        }
-      }
+    fireEvent.click(container.querySelector('#delete-1'));
```

Comment:
IMO we should be using `getByRole` to find the delete button. It should be correctly labelled and if it's not we should be doing it :)